File: Examples/Infovis/Python/hello_world.py
#!/usr/bin/env python

from vtk import *

source = vtkRandomGraphSource()

view = vtkGraphLayoutView()
view.AddRepresentationFromInputConnection(source.GetOutputPort())

view.GetRenderWindow().SetSize(600, 600)
view.ResetCamera()
view.Render()
view.GetInteractor().Start()

File: Examples/Infovis/Java/HelloWorld.java
import vtk.*;

public class HelloWorld {

  // Load VTK library and print which library was not properly loaded
  static {
    if (!vtkNativeLibrary.LoadAllNativeLibraries()) {
      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) {
        if (!lib.IsLoaded()) {
          System.out.println(lib.GetLibraryName() + " not loaded");
        }
      }
    }
    vtkNativeLibrary.DisableOutputWindow(null);
  }

  public static void main(String args[]) {
    vtkRandomGraphSource source = new vtkRandomGraphSource();

    vtkGraphLayoutView view = new vtkGraphLayoutView();
    view.AddRepresentationFromInputConnection(source.GetOutputPort());

    view.ResetCamera();
    view.Render();
    view.GetInteractor().Start();
  }
}
